rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java

```diff
@@ -416,7 +416,7 @@ public void shutDown() {
 
     @Override
     public String toString() {
-        return getDescription();
+        return this.getClass().getSimpleName() + ", nodeTimeout: [" + nodeTimeout + "], maxNodes: [" + maxNodes + "]";
```

Comment:
we should not put a comma after the infrastructure or policy name, simply a space

Reply:
OK. Applied it on HostsFileBasedInfrastructureManager as well for consistensy